components/brave_wallet/renderer/brave_wallet_js_handler.cc

```diff
@@ -439,6 +474,16 @@ v8::Local<v8::Promise> BraveWalletJSHandler::Request(
         std::move(tx_data), from,
         base::BindOnce(&OnAddUnapprovedTransaction, std::move(promise_resolver),
                        isolate, std::move(context_old), std::move(id)));
+  } else if (method == kEthSign || method == kPersonalSign) {
+    std::string address;
+    std::string message;
+    if (!ParseEthSignParams(normalized_json_request, &address, &message))
+      return v8::Local<v8::Promise>();
+
+    brave_wallet_provider_->SignMessage(
```

Comment:
Actually we should not auto sign for now, better to temporarily give an error here in case people enable this in Nightly before the confirm UI code is ready.

Reply:
addressed here https://github.com/brave/brave-core/pull/10433/commits/b5af31a927bf7f7932f6993724c774da0cf2051e